src/managers/translation_manager.ts

```diff
@@ -19,6 +19,9 @@ export class TranslationManager {
     public get_sms(key: string): string {
         return this.data["sms"]["he"]["nx_devices__" + key] || "N/A";
     }
+    public get_email(key: string): string {
```

Comment:
in "nx-translation" - I think instead of "nx_devices__camera_installation_html" we need to be more specific, like "nx_devices__camera_installation_minidash_body_html"
and use "body_html" instead of "html"